bitnami/seaweedfs/templates/volume/statefulset.yaml

```
@@ -0,0 +1,356 @@
+          startupProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.volume.startupProbe "enabled") "context" $) | nindent 12 }}
+            httpGet:
+              path: /status
+              port: http
+          {{- end }}
+          {{- end }}
+          {{- if .Values.volume.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.volume.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            {{- range .Values.volume.dataVolumes }}
+            - name: {{ .name }}
+              mountPath: {{ .mountPath }}
+              {{- if .subPath }}
+              subPath: {{ .subPath }}
+              {{- end }}
+            {{- end }}
+            - name: empty-dir
+              mountPath: /tmp
+              subPath: tmp-dir
+            {{- if or .Values.volume.config .Values.volume.existingConfigmap }}
+            - name: config
+              mountPath: /etc/seaweedfs/volume.toml
+              subPath: volume.toml
+              readOnly: true
+            {{- end }}
+            {{- if .Values.security.enabled }}
+            - name: security-config
+              mountPath: /etc/seaweedfs/security.toml
+              subPath: security.toml
+              readOnly: true
+            {{- if .Values.security.mTLS.enabled  }}
+            - name: ca-cert
+              readOnly: true
+              mountPath: /certs/ca
+            - name: master-cert
+              readOnly: true
+              mountPath: /certs/master
+            - name: volume-cert
+              readOnly: true
+              mountPath: /certs/volume
+            {{- if .Values.filer.enabled }}
+            - name: filer-cert
+              readOnly: true
+              mountPath: /certs/filer
+            {{- end }}
+            - name: client-cert
+              readOnly: true
+              mountPath: /certs/client
+            {{- end }}
+            {{- end }}
+          {{- if .Values.volume.extraVolumeMounts }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.volume.extraVolumeMounts "context" $) | nindent 12 }}
+          {{- end }}
+        {{- if .Values.volume.sidecars }}
+        {{- include "common.tplvalues.render" ( dict "value" .Values.volume.sidecars "context" $) | nindent 8 }}
+        {{- end }}
+      volumes:
+        - name: empty-dir
+          emptyDir: {}
+        {{- if or .Values.volume.config .Values.volume.existingConfigmap }}
+        - name: config
+          configMap:
+            name: {{ template "seaweedfs.volume.configmapName" . }}
+        {{- end }}
+        {{- if .Values.security.enabled }}
+        - name: security-config
+          configMap:
+            name: {{ printf "%s-security" (include "common.names.fullname" .) }}
+        {{- if .Values.security.mTLS.enabled  }}
+        - name: ca-cert
+          secret:
+            secretName: {{ template "seaweedfs.security.mTLS.caSecretName" . }}
```

We don't need to mount the CA private key in the containers:

```suggestion
            secretName: {{ template "seaweedfs.security.mTLS.caSecretName" . }}
            items:
              - key: tls.crt
                path: tls.crt
```